miscelanea/005-deploy-in-one-pod/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/freeipa/freeipa-server:centos-8
+
+ENV KRB5_TRACE=/dev/console
+
+# Workaround for running systemd mode from podman
+RUN mkdir /usr/sbin/original \
+    && ln -svf /usr/lib/systemd/systemd /usr/sbin/original/init \
+    && rm /usr/sbin/init
+COPY init-wrapper.sh /usr/sbin/init
+COPY init-data /usr/loca/sbin/init
```

Comment:
Review the path `/usr/loca/sbin/init`. Is this path right?

Reply:
It must be `/usr/local/sbin/init`

See: https://github.com/freeipa/freeipa-container/blob/master/Dockerfile.centos-8#L82

Reply:
Internal ticket: https://issues.redhat.com/browse/IDMOCP-187 . Resolving comment.